keyboards/handwired/3dp660/keymaps/default/rules.mk

```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
```

Comment:
```suggestion
```

Reply:
wouldn't it be better (If I don't need this line, which is the only line in the file) to just delete the file?

Reply:
There is also the `LTO_ENABLE` line below (although it won't actually do anything because `YES` is not a valid value). If you don't want/need LTO on the default keymap, go ahead and delete it.